src/enzo/ConvertParticles2ActiveParticles.C

```diff
@@ -0,0 +1,272 @@
+/***********************************************************************
+/
+/  Convert star objects to actve particles
+/
+/  written by: John Regan
+/  date:       May, 2018
+/  modified1:  
+/
+/  PURPOSE:
+/
+************************************************************************/
+#include "preincludes.h"
+
+#include <stdlib.h>
+#include <string.h>
+#include <stdio.h>
+ 
+#include "ErrorExceptions.h"
+#include "macros_and_parameters.h"
+#include "typedefs.h"
+#include "global_data.h"
+#include "Fluxes.h"
+#include "GridList.h"
+#include "ExternalBoundary.h"
+#include "Grid.h"
+#include "Hierarchy.h"
+#include "TopGridData.h"
+#ifdef TRANSFER
+#include "ImplicitProblemABC.h"
+#endif
+#include "CosmologyParameters.h"
+
+void CollectParticleTypes(char **active_particle_types, int numparticles);
```

Comment:
CollectParticleTypes and RebuildHierarchy aren't used below

Reply:
I've removed the RebuildHierarchy function and placed DEBUG statements around CollectParticleTypes.